src/test/java/pojo/Film.java

```diff
@@ -11,6 +13,16 @@ public class Film {
     private List<String> charactersHTTP;
     private String url;
 
+    public Film() {}
+
+    @JsonCreator
+    public Film(@JsonProperty("title") String title, @JsonProperty("characters") List<String> charactersHTTP, @JsonProperty("url") String url) {
```

Comment:
Could u please "beautify" the constructor arguments (not in the one line)? You can use the sonar or linter

Reply:
The code was modified